java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -650,7 +650,7 @@ public final float getFloat(long index) {
    * Get the value at index.
    */
   public final double getDouble(long index) {
-    assert type == DType.FLOAT64;
+    assert type == DType.FLOAT64 : type + " is not a supported double type.";
```

Comment:
Curious why the assert message was added only for this assert and not all the others (e.g.: getFloat, etc)?

Reply:
Sorry I was debugging something in the reduction tests, I added it to the others.